src/DotNet.Status.Web/appsettings.json

```diff
@@ -41,9 +41,14 @@
     "TableUri": "<<must override in environment>>",
     "TableSasToken": "[vault(deployment-table-sas-token)]"
   },
+  "WebHooks": {
     "github": {
       "SecretKey": {
-  } 
+        "default": "[vault(github-webhook-secret)]"
```

Comment:
This doesn't yet exist in the staging or prod vaults

Reply:
Oh! Correct. I haven't turned on the webhook yet.

Reply:
Added. :-)

Reply:
Turned it on and popped them in there.